src/Components/Breakdown.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Promise } from 'mongoose';
+class Breakdown extends Component {
+    constructor (){
+        super()
+        this.state={
```

Comment:
you should not start out with values here, you don't know what categories you will have